pkg/client/cache/store.go

```diff
@@ -116,6 +116,8 @@ type cache struct {
 	keyFunc KeyFunc
 }
 
+var _ Store = &cache{}
```

Comment:
hm, that leaves out `var ( ... )` blocks, but it probably doesn't matter if
the difference is that big. Never mind, then.

On Wed, Nov 11, 2015 at 3:50 PM, Paul Morie <notifications@github.com>
wrote:

> In pkg/client/cache/store.go
> <https://github.com/kubernetes/kubernetes/pull/17131#discussion_r44604621>
> :
>
> > @@ -116,6 +116,8 @@ type cache struct {
> >  	keyFunc KeyFunc
> >  }
> >
> > +var _ Store = &cache{}
>
> @lavalamp <https://github.com/lavalamp> looks like your style is in the
> minority from this:
>
> $ grep -nr 'var _ ' pkg/ | grep '(' | wc -l
> 36
>
> ★ pmorie:~/code/kubernetes (master)
> $ grep -nr 'var _ ' pkg/ | grep -v '(' | wc -l
> 113
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17131/files#r44604621>.
>